socialhome/streams/app/tests/components/AuthorBar.tests.js

```diff
@@ -0,0 +1,81 @@
+import "chai/register-should"
+import Vue from "vue"
+import BootstrapVue from "bootstrap-vue"
+import AuthorBar from "streams/app/components/AuthorBar.vue"
+
+
+Vue.use(BootstrapVue)
+
+describe("AuthorBar", () => {
+
+    describe("computed", () => {
+        describe("isUserRemote", () => {
+            it("should be the opposite of property isUserLocal", () => {
+                let propsData = {
```

Comment:
Yes, we can do that. I was also thinking about enabling JSX support in code and tests.